src/i18n/locales/nl.json

```diff
@@ -33,78 +33,78 @@
-    "principalRepresentatives": "Een account met een minimum van {{principalRepresentativeMinWeight}} NANO of >= 0,1% van het online stemgewicht dat eraan is gedelegeerd, is vereist om de status van hoofdvertegenwoordiger te krijgen. Wanneer geconfigureerd op een knooppunt dat aan het stemmen is, worden de stemmen die het produceert opnieuw uitgezonden door andere knooppunten naar wie ze ontvangen, waardoor het netwerk sneller consensus kan bereiken.",
-    "onlinePrincipalRepresentatives": "Online representatieve accounts die onlangs hebben gestemd.",
-    "lastTransaction": "Laatste verzonden transactie van een van de {{totalAccounts}} - accounts.",
+    "avgConfirmationTime": "De gemiddelde bevestigingstijd wordt lokaal op de node berekend. Als de node opnieuw wordt opgestart of in slechte staat verkeert, kan de gemiddelde bevestigingstijd langer zijn dan die van de rest van het netwerk.",
+    "circulatingSupply": "Het aantal munten dat in omloop is. Nano is volledig gedistribueerd, het aantal munten in omloop zal nooit toenemen.",
+    "transactionFees": "In tegenstelling tot andere cryptovaluta zijn nano transacties voor altijd kostenloos.",
+    "bitcoinTransactionFees": "Deze waarde heeft geen invloed op het NANO netwerk, het is uitsluitend ter vergelijking. Het vertegenwoordigt de som van de transactiekosten die Bitcoin gebruikers aan de miners betalen om hun transactie geaccepteerd te krijgen op het Bitcoin netwerk.",
+    "cps": "Aantal bevestigingen per seconde. Het aantal bevestigde blocks (verzenden of ontvangen) op het NANO netwerk. Het wordt elke minuut lokaal op de node gesampled. Als de node opnieuw wordt opgestart of in een verslechterde toestand verkeert, kan de CPS waarde hoger of lager zijn dan het werkelijke gemiddelde op het netwerk terwijl de node synchroniseert en opnieuw verbinding maakt met de andere nodes.",
+    "votingWeight": "Een account met een minimum van {{minWeight}} NANO of >= 0,1% van het online stemgewicht eraan gedelegeerd, is vereist om de status van hoofdvertegenwoordiger te krijgen. Indien geconfigureerd op een stemmende node, worden de stemmen die de node genereert opnieuw uitgezonden door andere nodes die de stemmen ontvangen, waardoor het netwerk sneller consensus kan bereiken.",
+    "pending": "Een transactiestatus waarbij een block dat geld verzendt, is gepubliceerd en bevestigd door het netwerk, maar een overeenkomend block dat het geld ontvangt, nog niet is verzonden of bevestigd.",
+    "nakamotoCoefficient": "De Nakamoto-coëfficiënt is een manier om de decentralisatie van een cryptovaluta of ander gedecentraliseerd systeem te kwantificeren. Het is het aantal entiteiten dat nodig is hebt om ten minste één essentieel subsysteem in gevaar te brengen. Voor Nano zou meer dan 67% van het online stemgewicht van hoofdvertegenwoordigers in gevaar moeten worden gebracht.",
```

Comment:
interesting, might have to change that one on other languages too, 50% -> 67%, will wait a few days tho to be sure it's the final direction its headed

Reply:
Yes I took the liberty to change it to 67% since that seems to be the current default as of 22.0.

Reply:
will use a variable in there in an upcoming PR